planck-cljs/test/planck/test_runner.cljs

```diff
@@ -16,14 +16,12 @@
    [planck.io-test]
    [planck.js-deps-test]
    [planck.repl-test]
-   [planck.shell-test]
-   [planck.socket.alpha-test]))
```

Comment:
Why not have updated tests?

Reply:
Cool. I've copied over the existing test namespace.